spec/services/disability_claim_service_spec.rb

```diff
@@ -3,8 +3,8 @@
 
 RSpec.describe DisabilityClaimService do
   let(:user) { User.sample_claimant }
-  let(:claim) do
-    FactoryGirl.create(:disability_claim, user_uuid: user.uuid, data: {})
+  let!(:claim) do
+    FactoryGirl.create(:disability_claim, user_uuid: user.uuid)
```

Comment:
Do you need an actual persisted claim to test failure? Would it be better to possibly do this create in a before(:all) so it does not write to the DB once for each spec?